.travis.yml

```diff
@@ -37,6 +37,12 @@ matrix:
     - php: 7.0
       env: PHPCS=1 DEFAULT=0
 
+    - php: 7.0
```

Comment:
Let's get one (7.0 then maybe) to work first, then lets think about adding more.